app/src/main/AndroidManifest.xml

```diff
@@ -12,13 +15,59 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".ui.MainActivity">
+
+        <activity android:name=".ui.mainActivity.MainActivity">
```

Comment:
Directory name should be related to the function/feature like `dashboard`, and if the text more than one word format should be `first_second`